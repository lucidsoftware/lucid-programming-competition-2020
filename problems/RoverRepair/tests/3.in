15
60
46
73
61
49
63
93
2
39
47
56
6
70
88
97
