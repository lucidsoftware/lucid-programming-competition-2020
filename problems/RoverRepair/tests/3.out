Swap nodes 6 and 61
